### PR TITLE
[Fix] Fix requirement about only ONNXRuntime without TensorRT

### PR DESCRIPTION
**Pull request by triple-Mu**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation for this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a potential influence on downstream projects, this PR should be tested with downstream projects, like MMDetection or MMClassification.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: projects/easydeploy/model/backendwrapper.py [hunk 1]
```diff
@@ -17,6 +17,7 @@
 
 
 class TRTWrapper(torch.nn.Module):
+    dtypeMapping = {}
```
Comment (PeterH0323) [inline projects/easydeploy/model/backendwrapper.py:20]:
```suggestion
    dtype_mapping = {}
```

File: projects/easydeploy/model/backendwrapper.py [hunk 2]
```diff
@@ -30,9 +31,19 @@ def __init__(self, weight: Union[str, Path],
         self.weight = weight
         self.device = device
         self.stream = torch.cuda.Stream(device=device)
+        self.__update_mapping()
         self.__init_engine()
         self.__init_bindings()
 
+    def __update_mapping(self):
+        self.dtypeMapping.update({
```
Comment (PeterH0323) [inline projects/easydeploy/model/backendwrapper.py:39]:
```suggestion
        self.dtype_mapping.update({
```

File: projects/easydeploy/model/backendwrapper.py [hunk 3]
```diff
@@ -71,14 +82,14 @@ def __init_bindings(self):
 
         for i, name in enumerate(self.input_names):
             assert self.model.get_binding_name(i) == name
-            dtype = self.torch_dtype_from_trt(self.model.get_binding_dtype(i))
+            dtype = self.dtypeMapping[self.model.get_binding_dtype(i)]
```
Comment (PeterH0323) [inline projects/easydeploy/model/backendwrapper.py:85]:
```suggestion
            dtype = self.dtype_mapping[self.model.get_binding_dtype(i)]
```

File: projects/easydeploy/model/backendwrapper.py [hunk 4]
```diff
@@ -71,14 +82,14 @@ def __init_bindings(self):
...
             shape = tuple(self.model.get_binding_shape(i))
             inputs_info.append(Binding(name, dtype, shape))
 
         for i, name in enumerate(self.output_names):
             i += self.num_inputs
             assert self.model.get_binding_name(i) == name
-            dtype = self.torch_dtype_from_trt(self.model.get_binding_dtype(i))
+            dtype = self.dtypeMapping[self.model.get_binding_dtype(i)]
```
Comment (PeterH0323) [inline projects/easydeploy/model/backendwrapper.py:92]:
```suggestion
            dtype = self.dtype_mapping[self.model.get_binding_dtype(i)]
```

**Comment by PeterH0323** [comment]
Link issue : https://github.com/open-mmlab/mmyolo/issues/468

**Review by PeterH0323 — APPROVED** [review]
LGTM

**Review by hhaAndroid — APPROVED** [review]

**Pull request merged**